test/CSModule.t.sol

```diff
@@ -3196,11 +3196,11 @@ contract CsmGetNodeOperatorNonWithdrawnKeys is CSMCommon {
         assertEq(keys, 2);
     }
 
-    function test_getNodeOperatorNonWithdrawnKeys_RevertWhenNoNodeOperator()
```

Comment:
it's worth to have the same for other methods where we removed the check

Reply:
fixed